src/webots/nodes/WbSolid.hpp

```diff
@@ -355,6 +355,7 @@ protected slots:
   bool mUseInertiaMatrix;  // indicates that the WbSolid uses the latest valid inertia matrix field for ODE physics computation
   WbVector3 mCenterOfMass;
   WbVector3 mScaledCenterOfMass;
+  const double referenceDensity = 1000.0;
```

Comment:
Thank you for this change!

Following our style guidelines (https://github.com/cyberbotics/webots/wiki/CPP-Coding-Style/#cs305-global-constant-names-including-enum-values-must-be-all-uppercase-using-underscore-to-separate-words), this should be changed to uppercase. It should also be moved to the `WbSolid.cpp` file close to the definition of `MASS_ZERO_THRESHOLD`.
```suggestion
  static const double REFERENCE_DENSITY = 1000.0;
```

Reply:
sure, i will make this change.